comp/core/tagger/taggerimpl/replay/tagger.go

```diff
@@ -69,6 +70,20 @@ func (t *Tagger) Tag(entityID types.EntityID, cardinality types.TagCardinality)
 	return tags, nil
 }
 
+// LegacyTag is an interface function that queries taggerclient singleton
```

Comment:
Same comment as above.

Actually in order to avoid repeating this comment in that many places, maybe we could just put the comment as:

```
// LegacyTag implements the LegacyTag method of the Tagger component
```

And then we can put the explanatory comment on the interface method directly.